source/update.rst

```diff
@@ -21,6 +28,27 @@ The following steps will allow you to update both of them to current version
 
    $ kubectl patch perconapgcluster/<cluster-name> --type json -p '[{"op": "replace", "path": "/spec/pause", "value": true},{"op":"replace","path":"/spec/pgBouncer/size","value":0}]'
 
+.. |rarr|   unicode:: U+02192 .. RIGHTWARDS ARROW
+
+#. If you upgrade the Operator **from a version earlier than 1.1.0**, the
+   following additional step is needed for the 1.0.0 |rarr| 1.1.0 upgrade.
+
+.. code:: bash
+
+   $ export CLUSTER=<cluster-name>
+   $ for user in postgres primaryuser $(kubectl get perconapgcluster/${CLUSTER} -o yaml | yq r - 'spec.user'); do args+="--from-literal=$user=$(kubectl get secret/${CLUSTER}-${user}-secret -o yaml | yq r - 'data.password' | base64 -d) "; done; eval kubectl create secret generic ${CLUSTER}-users-secret "${args}"
```

Comment:
```suggestion
   $ for user in postgres primaryuser $(kubectl get perconapgcluster/${CLUSTER} -o yaml | yq r - 'spec.user'); do args+="--from-literal=$user=$(kubectl get secret/${CLUSTER}-${user}-o yaml | yq r - 'data.password' | base64 -d) "; done; eval kubectl create secret generic ${CLUSTER}-users "${args}"
```
@cap1984  Am I right ?

Reply:
yeap. It's true for default CR configuration